script/build_docker.sh

```diff
@@ -47,4 +51,4 @@ fi
 
 echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin quay.io
```

Comment:
For docker_password and docker_username to work, we need to create env using DOCKER_PASSWORD and DOCKER_USERNAME secrets,  https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Reply:
ok, I'll pass them to env in workflow